desktop_version/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ IF(NOT WIN32)
 ENDIF()
 
 # Include Directories
-INCLUDE_DIRECTORIES(src tinyxml physfs lodepng)
+INCLUDE_DIRECTORIES(src tinyxml physfs lodepng utfcpp/source)
```

Comment:
PhysicsFS has a unicode system in it, can we use that instead of adding a very large dependency?

Reply:
PhysicsFS doesn't have a way to get the length of a UTF-8 string in codepoints.